src/main/java/me/dinowernli/grpc/prometheus/ClientMetrics.java

```diff
@@ -118,14 +132,24 @@ public void recordLatency(double latencySec) {
 
     Factory(Configuration configuration) {
       CollectorRegistry registry = configuration.getCollectorRegistry();
-      this.rpcStarted = rpcStartedBuilder.register(registry);
-      this.rpcCompleted = rpcCompletedBuilder.register(registry);
-      this.streamMessagesReceived = streamMessagesReceivedBuilder.register(registry);
-      this.streamMessagesSent = streamMessagesSentBuilder.register(registry);
+      this.labelHeaderKeys = metadataKeys(configuration.getLabelHeaders());
+      this.rpcStarted = rpcStartedBuilder
+              .labelNames(asArray(defaultRequestLabels, configuration.getSanitizedLabelHeaders()))
```

Comment:
Here too (and elsewhere in the PR), can we stick to the 4 space indent on break?

Reply:
Done